src/backend/services/users-api/src/app/routes.py

```diff
@@ -0,0 +1,114 @@
+from src.app import app
+from flask import request, jsonify
+from src.app import mongo
+import sys, os
+
+if os.environ.get('SERVICE_IN_DOCKER', False):
+    sys.path.append(os.path.abspath(os.path.join('..', '')))
+else:
+    sys.path.append(os.path.abspath(os.path.join('../..', '')))
+
+from shared import HttpCode, FailureReturnString, config
+
+
+@app.route("/")
+def index():
+    return jsonify(
+        status=True,
+        message="Hello Users of " + mongo.db.name
+    )
+
+
+@app.route("/login", methods=["POST"])
+def login():
+    data = request.get_json(force=True)
+
+    try:
+        username = data['username']
```

Comment:
Can you please change all uses of `username` to `umnetId` please

Reply:
I'm a little biased towards username just because this is in the context of the app and seems more general. @abhisachdev17  , can you weigh in on this too?